dynamic_rest/serializers.py

```diff
@@ -83,6 +84,16 @@ def __new__(cls, *args, **kwargs):
             meta = type('Meta', (), {})
             cls.Meta = meta
         meta.list_serializer_class = DynamicListSerializer
+
+        # Check for bulk resource creation with help of resource plural name.
+        # This should happen before actual constructor because class needs to
+        # be changed on DynamicListSerializer.
+        plural_name = cls.get_plural_name()
+        data = kwargs.get('data', fields.empty)
+        if data is not fields.empty and plural_name in data and len(data) == 1:
```

Comment:
Heh, not that simple, because you're getting loop because of `many_init`: https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/serializers.py#L104-L134
But I will consider this approach. It would be really nice to have.